src/infer.py

```diff
@@ -0,0 +1,111 @@
+from absl import flags, app
+import numpy as np
+import torch
+from imageio import imwrite
+from fridge import load_model
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string("checkpoint", None, "Path to checkpoint file.")
+flags.mark_flag_as_required("checkpoint")
+flags.DEFINE_string("numfonts", None, "Number of fonts to generate.")
+flags.mark_flag_as_required("numfonts")
+flags.DEFINE_string("path", None, "Path to save pngs to.")
+flags.mark_flag_as_required("path")
+
+CHARACTERS = [
+    'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O',
+    'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', 'a', 'b', 'c', 'd',
+    'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's',
+    't', 'u', 'v', 'w', 'x', 'y', 'z', 'zero', 'one', 'two', 'three', 'four',
+    'five', 'six', 'seven', 'eight', 'nine', 'exclam', 'numbersign', 'dollar',
+    'percent', 'ampersand', 'asterisk', 'question', 'at'
+]
+
+DEVICE = torch.device("cuda:1" if torch.cuda.is_available() else "cpu")
+
+
+def rasterize(x, resolution=64, sigma=0.01):
```

Comment:
Is there a reason this isn't' being imported from elsewhere?

Reply:
I'd have to import a class to do it. tbh it's probably worth moving out into a `utils.py`, along with `CHARACTERS` and other things